source/draw/RasterYuv420Semiplanar.ooc

```diff
@@ -249,4 +251,5 @@ RasterYuv420Semiplanar: class extends RasterYuvSemiplanar {
 		fileWriter write(this uv buffer pointer as Char*, this uv buffer size)
 		fileWriter close()
 	}
+	createPaintEngine: override func -> PaintEngine { Yuv420PaintEngine new(this) }
```

Comment:
Same as `RasterUv` - I didn't need it.
I created implementation only for classes possibly needed for the windows port. Also, it keeps the pull request smaller :)